### DISCUSSION TITLE
Dashboards grouping

### DISCUSSION THREAD

**Discussion by vishwa-ims** [RevealBi/Reveal.Sdk#257] [ASP.NET Core]
Hi ,
I just wanted to check if you have Dashboard grouping feature readily available . We are trying to provide access to specific dashboards to specific group of users . Just wanted to check if you have any support for this

**Comment by brianlagunas** [comment]
This would be application code.  Reveal is just an SDK to create, edit, and load dashboards. You write the logic to determine how those dashboards are saved and surfaced to your user.
